client/scala/src/main/scala/ai/verta/swagger/client/HttpClient.scala

```diff
@@ -71,7 +71,27 @@ class HttpClient(val host: String, val headers: Map[String, String]) {
     })
   }
 
+  /** Make a (raw) request, returning the body of the response
+   *  @param method method of the request
+   *  @param url url of the request
+   *  @param query query parameters
+   *  @param localHeaders local headers
+   *  @param body body of the request
+   *  @return body of the response
+   */
   def requestRaw(method: String, url: String, query: Map[String, List[String]], localHeaders: Map[String, String], body: InputStream)(implicit ec: ExecutionContext) = {
```

Comment:
Why was this sprint? I think it makes more sense to return a case class with the response+headers, no? It seems weird to have different functions, one to fetch body, one to fetch the headers.

Reply:
Total didn't think about that. Fixed.